Website/Packages/datax-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
looks straightforward enough. can @barrytu or @dennygu review?  do we need to rev the version\n